### PR TITLE
Functional Tests: `GET /api/languages`

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Functional Tests: `GET /api/languages`
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is :green_circle: 
| UI Tests          | https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/8570923094
| Fixed issue or discussion?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

## Linked Issues : 
* https://github.com/PrestaShop/PrestaShop/issues/35860

### CODE REVIEW

File: tests/UI/campaigns/functional/API/02_endpoints/05_language/01_getAPILanguages.ts [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+      await testContext.addContextItem(this, 'testIdentifier', 'resetFilter', baseContext);
+
+      await languagesPage.resetFilter(page);
+
+      const numberOfLanguages = await languagesPage.resetAndGetNumberOfLines(page);
+      expect(numberOfLanguages).to.be.above(0);
+    });
+  });
+
+  // Pre-condition: Create an API Client
```
Comment (boubkerbribri) [inline tests/UI/campaigns/functional/API/02_endpoints/05_language/01_getAPILanguages.ts]:
```suggestion
  // Post-condition: Delete the API Client
```

**Review by boubkerbribri — DISMISSED** [review]
One small comment but LGTM

File: tests/UI/campaigns/functional/API/02_endpoints/05_language/01_getAPILanguages.ts (hunk 1 above)
Comment (Progi1984) [reply tests/UI/campaigns/functional/API/02_endpoints/05_language/01_getAPILanguages.ts]: Fixed

Comment (Progi1984) [reply tests/UI/campaigns/functional/API/02_endpoints/05_language/01_getAPILanguages.ts]: (Thanks boss)

**Review by jolelievre — APPROVED** [review]
Thanks @Progi1984

**Review by nesrineabdmouleh — APPROVED** [review]

**Pull request merged**